src/plugincode/format_csv.py

```diff
@@ -0,0 +1,232 @@
+#
```

Comment:
@pombredanne What about `formats.py`? The plugin which returns the `--format` option, should it be moved to `formattedcode` as well?